zerver/actions/message_edit.py

```diff
@@ -1003,6 +1031,13 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
                     visibility_policy=new_visibility_policy,
                 )
 
+    if topic_name is not None and not target_topic_has_messages:
+        first_message_moved = changed_messages.first()
+        if first_message_moved:
+            apply_automatic_unmute_follow_topics_policy(
+                user_profile, first_message_moved, target_stream, target_topic_name
+            )
```

Comment:
I didn't make an if condition to verify if it is a bot or not before applying `apply_automatic_unmute_follow_topics_policy`, assuming no one would move a bot's message to another topic. Is it needed?

Reply:
That assumption doesn't make any sense, of course any message can be moved.